tests/_server/api/endpoints/test_health.py

```diff
@@ -68,6 +68,9 @@ def test_memory(client: TestClient) -> None:
     computer = response.json()["kernel"]
     assert computer["memory"] is None
 
+    gpu = response.json()["gpu"]
+    assert len(gpu) == 0
```

Comment:
```suggestion
    assert "gpu" in response.json(), "Testing machine specs may differ"
```

Reply:
i don't think people are running tests in gpus. i'd prefer 0.

if they are, i'd like the test to fail so it can be improved to _actually_ test the response

Reply:
I run tests on my tower :'( but it's still in Maryland